### PR TITLE
comp added : p1- rating, toast , grid with func

### PR DESCRIPTION
**Pull request by manojsarna**
Components Added
1. Rating - with JS Functionality
2. Toast - with Js Functionality
3. Grid 

some minor changes , cards css and some layout changes throughout the code.

Usage has been update

### CODE REVIEW

File: lib/Components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+.grid-30-70{
+    display: grid;
+    grid-template-columns: 30% 70%;  
+    justify-content: center;
+    align-content: center;
+}
+
+.grid-4c{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (zoef-yt) [inline lib/Components/Grid/grid.css:19]: repeat(4,1fr)

File: lib/Components/Navigation/navigation.css [hunk 2]
```diff
@@ -66,7 +66,7 @@
     position: relative;
     border-radius:5px;
     padding:1rem 1.5rem;
-    transition: all ease-in 0.1s;
+    /* transition: all ease-in 0.1s; */
```
Comment (zoef-yt) [inline lib/Components/Navigation/navigation.css:69]: remove commented code

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by j24m — APPROVED** [review]
LGTM !!

File: lib/Components/Grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+.sm-grid-card-hor{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    border-radius: 5px;
+    
+
```
Comment (b0llu) [inline lib/Components/Grid/grid.css:47]: remove extra spacing

**Review by b0llu — APPROVED** [review]
Great UI you can go ahead and merge

File: lib/Components/Grid/grid.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    place-items: center;
+}
+
+.sm-grid-card-hor{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    border-radius: 5px;
+
```
Comment (gauravsinhaweb) [inline lib/Components/Grid/grid.css:46]: better if you use common class for repetitive CSS property  (grid)

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: lib/Components/Grid/grid.css (hunk 4 above)
Comment (gauravsinhaweb) [reply lib/Components/Grid/grid.css:46]: Nice work!

File: lib/Components/Grid/grid.css [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    gap:1rem;
+    width: 100%;
+    min-width: 50rem;
+    margin: 0 auto;
+}
+
+.grid-4c .sm-card{
+    width:fit-content;
```
Comment (yash9263) [inline lib/Components/Grid/grid.css:27]: Need refactoring

File: lib/Components/Grid/grid.css [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    max-width: fit-content;
+
+}
+.grid-content-center{
+    justify-content: center;
+    align-items:center;
+}
```
Comment (yash9263) [inline lib/Components/Grid/grid.css:34]: can use place-items here

File: lib/Components/Rating/rating.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
+const allRatings = document.querySelectorAll(".sm-rating .fas");
```
Comment (yash9263) [inline lib/Components/Rating/rating.js:1]: Great work nice approach

File: lib/Components/Grid/grid.css (hunk 5 above)
Comment (manojsarna) [reply lib/Components/Grid/grid.css:27]: will check

File: lib/Components/Grid/grid.css (hunk 6 above)
Comment (manojsarna) [reply lib/Components/Grid/grid.css:34]: yes will do

**Pull request merged**